types/address.go

```diff
@@ -256,6 +264,31 @@ func (aa AccAddress) Format(s fmt.State, verb rune) {
 	}
 }
 
+// LengthPrefixedAddress prefixes the address bytes with its length, this is used
+// for variable-length components in store keys. Note: All addresses should be
+// max 255 bytes, or else this function panics.
```

Comment:
comment must be updated: the  function doesn't panics, and mention that 0 lenght address is invalid as well.